cmd/log-shuttle/main.go

```diff
@@ -66,6 +72,15 @@ func ParseFlags(c shuttle.Config) shuttle.Config {
 
 	flag.Parse()
 
+	if skipHeaders {
+		log.Println("Warning: Use of -skip-headers is deprecated, use -input-format=2 (rfc5424) instead")
```

Comment:
Done. It maps to ints internally, so no real change to code outside of cmd/log-shuttle/main.go